packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -310,6 +310,8 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
         return defaultOnRender(inlineImage);
       }
 
+      const pointerEvents = inlineImage.imageAttributes.src === '' ? 'none' : 'unset';
```

Comment:
how about the dummy url we are using to render broken image icon? Do we want to make it none clickable as well?

Reply:
another question for future us, maybe this is how we can re-try fetching-images? @JoshuaLai like you click on broken image again to fetch again